packages/react/src/components/Dashboard/DashboardGrid.test.e2e.jsx

```diff
@@ -174,7 +174,7 @@ const DashboardAllCardsAsResizable = ({ breakpoint, type }) => {
     const hasLinks = index === 4;
     const hasSort = index === 1;
     const hasThresholds = index === 2;
-    const hasExpandedRows = index === 3;
+    const hasExpandedRows = index === 5;
```

Comment:
Not following why this change is along with the card size changes?

Reply:
This is due to the added sizes "SMALLTHICK", "MEDIUMTHICK", "LARGETHICK". The index is base on the index in CARD_SIZES. `const hasExpandedRows = index === 3;` was supposed to work with `LARGETHIN` in the test. The new index for `LARGETHIN` is 5.  

I updated the test to omit "SMALLTHICK", "MEDIUMTHICK" for now so that we don't have to change the index here anymore.